lib/pipeline/addon/components/settings-git/component.js

```diff
@@ -85,7 +85,7 @@ export default Component.extend({
       ]).then(() => {
         cb(true);
       }).catch((err) => {
-        get(this, 'growl').fromError('Error saving limit and reqeust', err);
+        get(this, 'growl').fromError('Error saving limit and request', err);
```

Comment:
Can you make this a translation as well? I can't find any reason why it would not be.